openedx/core/djangoapps/user_authn/api/views.py

```diff
@@ -0,0 +1,56 @@
+"""
+Logistration API Views
+"""
+
+from django.conf import settings
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.throttling import AnonRateThrottle
+from rest_framework.views import APIView
+
+from openedx.core.djangoapps.user_authn.utils import third_party_auth_context
+
+REDIRECT_KEY = 'redirect_to'
+
+
+class ThirdPartyAuthContextThrottle(AnonRateThrottle):
+    """
+    Setting rate limit for ThirdPartyAuthContext API
+    """
+    rate = settings.LOGISTRATION_API_RATELIMIT
+
+
+class ThirdPartyAuthContext(APIView):
```

Comment:
nit: Can we add `View` at the end of the class name? Or change it to something like this `TPAContextView` if that looks too long?